cmd/restic/cmd_check.go

```diff
@@ -329,7 +330,9 @@ func selectRandomPacksByPercentage(allPacks map[restic.ID]int64, percentage floa
 	if packsToCheck < 1 {
 		packsToCheck = 1
 	}
-	idx := rand.Perm(packCount)
+	timeNs := time.Now().UTC().UnixNano()
```

Comment:
I'd prefer to use `time.Now().UnixNano()` here, which is also already used in a few test. The conversion to UTC doesn't have any effect here.